3-reveal/migrations/5-IRS/3-Namibia-2019/verify/namibia_focus_area_irs.psql

```diff
@@ -14,7 +14,6 @@ SELECT
     jurisdiction_id,
     jurisdiction_parent_id,
     jurisdiction_name,
-    jurisdiction_geometry,
```

Comment:
Is this safe / necessary?  We'll always need to rebuild via the old version if we're using migrations, no?

Reply:
sadly it will break the tests if we leave it :(